src/Manager/CanvasHolder.ts

```diff
@@ -0,0 +1,5 @@
+import Canvas from '../Canvas';
+
+export default interface CanvasHolder {
+  canvas: Canvas;
+}
```

Comment:
La comme ca, je comprend pas à quoi ca sert...
Par rapport a directement utiliser le type CanvasRenderingContext2D ? D'ailleurs c'est quoi comme type _Canvas_ ? Ca existe ?

Reply:
So this is a bit tricky, indeed.

Let's take a look at the Shape class: